### DISCUSSION TITLE
Generate records out of OpenApi definition (yaml)

### DISCUSSION THREAD

**Discussion by mauriciomagni** [akhansari/FSharp.OpenApi#11] [General]
Hello
Is it possible to use this tool to generate f# records?
Thanks in advance

**Comment by akhansari** [comment]
Hi Mauricio,

This project just provides some computation expressions for [OpenAPI.NET](https://github.com/microsoft/OpenAPI.NET) and a little [helper](https://github.com/akhansari/FSharp.OpenApi/blob/master/OpenApi/Factory.fs) to generate the json definition.

You can use [SwaggerProvider](https://github.com/fsprojects/SwaggerProvider) to smoothly generate models from the definitions.

**Reply by mauriciomagni** [reply]
thanks!
